.gitlab-ci.yml

```diff
@@ -89,11 +89,15 @@ trigger:mender-dist-packages:
   stage: trigger
   before_script:
     - apk add --no-cache curl
+    - export MENDER_CLIENT_VERSION=$(git tag | egrep -e '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -V -r | head -n1)
+    # This will only be true in the case that we have a new newest client tag
+    - test ${MENDER_CLIENT_VERSION} = ${CI_COMMIT_TAG} && export PUBLISH_LATEST_CLIENT_PACKAGE="true" || true
```

Comment:
I would set a default value of "false", adding:
```
    - PUBLISH_LATEST_CLIENT_PACKAGE="false"
```

I am not sure if this is important, will curl fail parsing a parameter like `-F variables[something]=` ?

Reply:
Better safe than sorry. Adding.